### PR TITLE
[Docs](bangc-ops):Fixed doc problems

### PR DESCRIPTION
**Pull request by GraceGuanH**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

Please describe your motivation and the goal you want to achieve through this pull request.

## 2. Modification

Please briefly describe what modification is made in this pull request, and indicate where to make the modification.

## 3. Test Report

If you want to know how to do operator testing, you can see [GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).

#### 3.1.1 Accuracy Acceptance Standard

For static threshold standard details, see: [MLU-OPS Accuracy Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Accuracy-Acceptance-Standard.md).

- [ ] diff1: diff1 <= 3e-3
- [ ] diff2: diff2 <= 3e-3

#### 3.1.2 Operator Scheme checklist

|     No.        |                 Details              |            Check Results             |
|----------------|--------------------------------------|--------------------------------------|
|        1       |Supported hardware                    |             MLU370<br>MLU590         |
|        2       |Job types                             |          block <br> U1 <br> U4       |
|        3       |Layouts                               |          NHWC 、NCHW、ARRAY etc      |
|        4       |Whether multi-dimensions are supported|                                      |
|        5       |Whether element zero is supported     |                                      |
|        6       |Data type(half/float)                 |           half / float etc           |
|        7       |Whether there is size limit           |                                      |

#### 3.1.3 New Feature Test

If you have checked the following items, please tick the relevant box.

- [ ] Data type test
- [ ] Multi-dimensional tensor test
- [ ] Layout test
- [ ] Different size/integer remainder end segment/alignment misalignment test
- [ ] Zero dimensional tensor test/zero element test
- [ ] stability test
- [ ] Multiple platform test
- [ ] Gen_case module test
- [ ] Nan/INF tests 
- [ ] Bug fix tests
- [ ] For memory leak check details, see[GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).
- [ ] For code coverage check details, see: [GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).
- [ ] For I/O calculation efficiency check details, see: [MLU-OPS Performance Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Performance-Acceptance-Standard.md).

#### 3.1.4 Parameter Check

When a new operator is submitted, the test points are given and the test results are stated.

|                   Test Point                    | Acceptance Standard | Test Result (Error Message) |
| ----------------------------------------------- | --------------------| --------------------------- |
| Whether it conforms to the operator restriction |     Normal error    |                             |
| Whether illegal parameters are passed           |     Normal error    |                             |

### 3.2 Accuracy Test

For the cases used in the New Feature Test section, the features and the number of cases are recorded here. When multiple operations are tested, multiple tables are needed to include details of these operations.

Operation:

|Test Point           | Description                      | Quantity |  Comment |
|----------           |----------------------------------|----------|  --------|
|Data type test       |half/float/int8                   |          |          |
|Mult-tensor test     |Supports 1-8 dims                 |          |          |
|Layout test          |Supports NCHW/NHWC                |          |          |
|Zero element test    |Whether to support this test      |          |          |
|Stability test       |--gtest_repeat=NUM<br>--thread=NUM|          |          |
|Mult-platform test   |MLU370/MLU590                     |          |          |
|Nan/INF test         |Whether to support this test      |          |          |
|Memory leak check    |Test result                       |          |          |
|Code coverage check  |Test result                       |          |          |

### 3.3 Performance Test

See [MLU-OPS Performance Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Performance-Acceptance-Standard.md) for details.

Platform：MLU370

|Operation|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-------|----|----|-----|----|----|----|-----|
|op_name|    |    |     |    |    |    |     |
|op_name|    |    |     |    |    |    |     |

Platform：MLU590

|Operation|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-------|----|----|----|----|----|----|-----|
|op_name|    |    |    |    |    |    |     |
|op_name|    |    |    |    |    |    |     |

### 3.4 Summary Analysis

Please give a brief overview here, if you want to note and summarize the content.

### CODE REVIEW

File: bangc-ops/mlu_op.h [hunk 1]
```diff
@@ -470,7 +470,7 @@ mluOpUpdateContextInformation(mluOpHandle_t handle);
 // Group:Runtime Management
 /*!
  * @brief Releases the resources of the specified MLUOP handle \b handle that was
- * created by the ::mluOpCreate function. It is usually the last call to destroy
+ * created by ::mluOpCreate. It is usually the last call to destroy
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:473]: 需要用tool/format 刷一下整体格式

File: bangc-ops/mlu_op.h [hunk 2]
```diff
@@ -6688,13 +6683,13 @@ mluOpPsamaskForward(mluOpHandle_t handle,
 // Group:Psamask
 /*!
  * @brief Computes the gradients of input tensor \b dx with the gradients of output tensor \b dy
- * according to \b h_mask , \b w_mask and \b psa_type.
+ * according to \b h_mask , \b w_mask, and \b psa_type.
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:6686]: 为什么 h_mask 后的逗号前跟一个空格，w_mask后却没有？

很多处修改是逗号前跟一个空格，其他没修改，这里是什么原因？

**Review by tudejiang79 — APPROVED** [review]

File: bangc-ops/mlu_op.h [hunk 3]
```diff
@@ -10745,24 +10739,24 @@ mluOpGetMoeDispatchBackwardGateWorkspaceSize(mluOpHandle_t handle,
...
  *
  * @par Scale Limitation
- * - The first dimension of \b indices tensor, \b locations tensor, \b input tensor and \b grad_gates
+ * - The first dimension of \b indices tensor , \b locations tensor , \b input tensor , and \b grad_gates
  *   tensor must be the same size and equal to \b samples.
  * - The second dimension of \b input tensor and \b dispatch tensor must be equal to \b hidden.
  * - The first dimension of \b dispatch tensor must be equal to the multiplication result of
  *   the \b capacity and \b num_experts.
- * - The value of the input parameters \b samples, \b capacity, \b hidden and \b num_experts
+ * - The value of the input parameters \b samples , \b capacity , \b hidden , and \b num_experts
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:10750]: 比如这里

File: bangc-ops/mlu_op.h [hunk 4]
```diff
@@ -4713,17 +4709,17 @@ mluOpRotatedFeatureAlignForward(const mluOpHandle_t handle_,
...
  * Handle to an MLUOP context that is used to manage MLU devices and queues in
  * ::mluOpRotatedFeatureAlignBackward operation. For detailed information, see ::mluOpHandle_t.
  * @param[in] top_output_desc
- * The descriptor of the top_output tensor. For detailed information, see ::mluOpTensorDescriptor_t.
+ * The descriptor of the tensor \b top_output. For detailed information, see ::mluOpTensorDescriptor_t.
  * @param[in] top_output
  * Pointer to the MLU memory that stores the top_output tensor.
  * @param[in] bboxes_desc
- * The descriptor of bboxes, which contains the dimension and layout of bboxes tensor. For detailed
+ * The descriptor of the tensor \b bboxes, which contains the dimension and layout of bboxes tensor. For detailed
```
Comment (tudejiang79) [inline bangc-ops/mlu_op.h:4722]: 这里的detailed怎么出来了 -> *  detailed

File: bangc-ops/mlu_op.h [hunk 5]
```diff
@@ -4732,7 +4728,7 @@ mluOpRotatedFeatureAlignForward(const mluOpHandle_t handle_,
  * @param[in] points
  * An int value that is the number of sample points. Only 1 and 5 are supported. The default value is 1.
  * @param[in] bottom_input_desc
- * The descriptor of bottom_input tensor, which contains the dimension and layout of bottom_input tensor.
+ * The descriptor of the tensor \b bottom_input, which contains the dimension and layout of bottom_input tensor.
```
Comment (tudejiang79) [inline bangc-ops/mlu_op.h:4731]: * tensor

File: bangc-ops/mlu_op.h [hunk 6]
```diff
@@ -5304,8 +5300,8 @@ mluOpYoloBox(mluOpHandle_t handle,
  * - None.
  *
  * @par Note
- * - The operation does not support MLU200 series.
- * - You need to set the initial value for the output \b pos_memo before calling the operation, and initialize it to a
+ * - The function does not support MLU200 series.
+ * - You need to set the initial value for the output \b pos_memo before calling the funtion, and initialize it to a
```
Comment (tudejiang79) [inline bangc-ops/mlu_op.h:5304]: * it to a

File: bangc-ops/mlu_op.h [hunk 7]
```diff
@@ -5304,8 +5300,8 @@ mluOpYoloBox(mluOpHandle_t handle,
...
  *   negative number.
```
Comment (tudejiang79) [inline bangc-ops/mlu_op.h:5305]: 这行与上一行合并

File: bangc-ops/mlu_op.h [hunk 8]
```diff
@@ -6563,10 +6558,10 @@ mluOpRoiawarePool3dForward(mluOpHandle_t handle,
  * @par Data Type
  * - This function supports the following data types for input tensor \b pts_idx_of_voxels , \b argmax , \b grad_out
  *   and output tensor \b grad_in .
```
Comment (tudejiang79) [inline bangc-ops/mlu_op.h:6560]: * grad_out

File: bangc-ops/mlu_op.h [hunk 9]
```diff
@@ -6641,14 +6636,14 @@ mluOpRoiawarePool3dBackward(mluOpHandle_t handle,
  *   - y: float
  *
  * @par Data Layout
- * - The supported data layouts of input tensor \b x and output tensor \b y are as follows
+ * - The supported data layouts of input tensor \b x and output tensor \b y are as follows:
  *  - x: NHWC
  *  - y: NHWC
```
Comment (tudejiang79) [inline bangc-ops/mlu_op.h:6641]: 以上两行空一格

File: bangc-ops/mlu_op.h [hunk 10]
```diff
@@ -6641,14 +6636,14 @@ mluOpRoiawarePool3dBackward(mluOpHandle_t handle,
...
  *
  * @par Scale Limitation
  * - The shape of \b x must be [N, H, W, C].
  * - The shape of \b y must be [N, H, W, C].
- * - All dimension size of \b x and \b y must be the same, except the C dimension.
+ * - All dimension sizes of \b x and \b y must be the same, except the C dimension.
  * - If the shape of \b x is set to [N, H, W, C], the size of C dimension should be \b h_mask * \b
  *   w_mask .
```
Comment (tudejiang79) [inline bangc-ops/mlu_op.h:6648]:
```suggestion
 *   w_mask.
```

File: bangc-ops/mlu_op.h [hunk 11]
```diff
@@ -8213,8 +8208,8 @@ mluOpScatterNd(mluOpHandle_t handle,
 /*!
  * @brief Distributes the elements in tensor \b updates to tensor \b output according to the coordinates
  * in tensor \b indices. Compared with ::mluOpScatterNd, this function supports the \b mode parameter that contains
- * more calculation methods when \b indices contains duplicates. This operation is the inverse of the ::mluOpGatherNd
- * operation which extracts values or slices from a given tensor.
+ * more calculation methods when \b indices contains duplicates. This operation is the inverse of ::mluOpGatherNd
```
Comment (tudejiang79) [inline bangc-ops/mlu_op.h:8211]: \* ::mluOpGatherNd

File: bangc-ops/mlu_op.h (hunk 1 above)
Comment (GraceGuanH) [reply bangc-ops/mlu_op.h:473]: 已刷

File: bangc-ops/mlu_op.h (hunk 9 above)
Comment (GraceGuanH) [reply bangc-ops/mlu_op.h:6641]: 已改

File: bangc-ops/mlu_op.h (hunk 10 above)
Comment (GraceGuanH) [reply bangc-ops/mlu_op.h:6648]: 已改

File: bangc-ops/mlu_op.h (hunk 8 above)
Comment (GraceGuanH) [reply bangc-ops/mlu_op.h:6560]: 已改

File: bangc-ops/mlu_op.h (hunk 3 above)
Comment (GraceGuanH) [reply bangc-ops/mlu_op.h:10750]: 已改

File: bangc-ops/mlu_op.h (hunk 2 above)
Comment (GraceGuanH) [reply bangc-ops/mlu_op.h:6686]: 已改

File: bangc-ops/mlu_op.h [hunk 12]
```diff
@@ -10745,28 +10744,28 @@ mluOpGetMoeDispatchBackwardGateWorkspaceSize(mluOpHandle_t handle,
...
  * @par API Dependency
- * - Before calling this function to perform ::mluOpMoeDispatchBackwardGate operation, you need to get
- *   the size of workspace by the ::mluOpGetMoeDispatchBackwardGateWorkspaceSize function.
+ * - Before calling this function to perform ::mluOpMoeDispatchBackwardGate, you need to get
+ *   the size of workspace by ::mluOpGetMoeDispatchBackwardGateWorkspaceSize.
  *
  * @par Note
  * - This function is only supported on MLU300 series or above platforms.
- * - The parameter \b samples , \b capacity , \b hidden and \b num_experts should not be negative.
+ * - The parameter \b samples , \b capacity , \b hidden , and \b num_experts should not be negative.
```
Comment (tudejiang79) [inline bangc-ops/mlu_op.h:10768]: 逗号前有空格

File: bangc-ops/mlu_op.h [hunk 13]
```diff
@@ -10694,27 +10693,27 @@ mluOpGetMoeDispatchBackwardGateWorkspaceSize(mluOpHandle_t handle,
...
+ * Pointer to the MLU memory that stores the input tensor.
  * @param[in] dispatch_desc
- * The descriptor of input tensor \b dispatch, which contains dimension, data type and data layout.
+ * The descriptor of the tensor \b dispatch, which contains dimension, data type, and data layout.
  * For detailed information, see ::mluOpTensorDescriptor_t.
  * @param[in] dispatch
  * Pointer to the MLU memory that stores the \b dispatch tensor.
  * @param[in] samples
- * The number of elements in the \b indices tensor, \b locations tensor and \b grad_gates tensor.
+ * The number of elements in the \b indices tensor , \b locations tensor , and \b grad_gates tensor.
```
Comment (tudejiang79) [inline bangc-ops/mlu_op.h:10716]: 逗号前空格

**Review by WangYuan-cn — APPROVED** [review]

File: bangc-ops/mlu_op.h [hunk 14]
```diff
@@ -10745,28 +10744,28 @@ mluOpGetMoeDispatchBackwardGateWorkspaceSize(mluOpHandle_t handle,
  *   - grad_gates tensor: float
  *
  * @par Data Layout
- * - The supported layout of input and output tensors must be \p MLUOP_LAYOUT_ARRAY.
+ * - The supported layout of the input tensors and output tensors must be \p MLUOP_LAYOUT_ARRAY.
  *
  * @par Scale Limitation
- * - The first dimension of \b indices tensor, \b locations tensor, \b input tensor and \b grad_gates
+ * - The first dimension of \b indices tensor , \b locations tensor , \b input tensor , and \b grad_gates
```
Comment (tudejiang79) [inline bangc-ops/mlu_op.h:10750]: 这个的逗号要删除

File: bangc-ops/mlu_op.h (hunk 12 above)
Comment (wushaoqiang2168) [inline bangc-ops/mlu_op.h:10768]: parameters

File: bangc-ops/mlu_op.h [hunk 15]
```diff
@@ -10579,7 +10578,7 @@ mluOpIndiceConvolutionForward(mluOpHandle_t handle,
  *
  * @par Data Type
  * - This function supports the following data types for input tensor \b gates , \b indices ,
- *   \b locations , \b input , \b dispatch .
+ *   \b locations , \b input , and \b dispatch .
```
Comment (wushaoqiang2168) [inline bangc-ops/mlu_op.h:10581]: 改成下面这个描述，会不会更好些(与backward_gate保持一致)？
* - The supported data types of input and output tensors are as follows:
 *   - gates tensor: float
 *   - indices tensor: int32
 *   - locations tensor: int32
 *   - input tensor: float
 *   - dispatch tensor: float

File: bangc-ops/mlu_op.h [hunk 16]
```diff
@@ -10465,10 +10463,11 @@ mluOpGetIndiceConvolutionForwardWorkspaceSize(mluOpHandle_t handle,
  *
  * @par Data Type
  * - This function supports the combination of the following data types:
- *   - input tensor \b features, \b filters, \b indice_pairs and output tensor \b features_out: half, half, int32, half
- *   - input tensor \b features, \b filters, \b indice_pairs and output tensor \b features_out: float, float, int32,
- *     float.
- * - The supported data type of array \b indice_num, scalar \b inverse and \b sub_m is int64.
+ *   - input tensor \b features , \b filters , \b indice_pairs , and output tensor \b features_out: half, half, int32,
```
Comment (wushaoqiang2168) [inline bangc-ops/mlu_op.h:10466]: input tensors ?

File: bangc-ops/mlu_op.h [hunk 17]
```diff
@@ -9733,25 +9730,25 @@ mluOpGetIndiceConvolutionBackwardDataWorkspaceSize(mluOpHandle_t handle,
  *
  * @param[in] handle
  * Handle to an MLUOP context that is used to manage MLU devices and queues in
- * the ::mluOpIndiceConvolutionBackwardData operation. For detailed
+ * ::mluOpIndiceConvolutionBackwardData. For detailed
  * information, see ::mluOpHandle_t.
  * @param[in] output_grad_desc
  * The descriptor of input data \b output_grad, which contains dimension, data
  * type and data layout.
```
Comment (wushaoqiang2168) [inline bangc-ops/mlu_op.h:9737]: data type, and data layout

File: bangc-ops/mlu_op.h [hunk 18]
```diff
@@ -9761,14 +9758,14 @@ mluOpGetIndiceConvolutionBackwardDataWorkspaceSize(mluOpHandle_t handle,
...
  * workspace, see "Cambricon BANGC OPS User Guide".
  * @param[in] workspace_size
- * The size of the extra workspace in bytes that is used in the
- * ::mluOpIndiceConvolutionBackwardData operation.
+ * The size of the extra workspace in bytes that is used in
+ * ::mluOpIndiceConvolutionBackwardData.
  * @param[in] input_grad_desc
- * The descriptor of output data \b input_grad, which contains dimension, data
+ * The descriptor of the output data \b input_grad, which contains dimension, data
  * type and data layout.
```
Comment (wushaoqiang2168) [inline bangc-ops/mlu_op.h:9769]: data type ,

Comment (GraceGuanH) [reply bangc-ops/mlu_op.h:9753]: 已改

**Review by duzekunKTH — APPROVED** [review]

**Pull request merged**